pkg/netdevice/vdpa.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2020 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package netdevice
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+
+	vdpa "github.com/redhat-virtio-net/govdpa/pkg/kvdpa"
+
+	pluginapi "k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+
+	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/types"
+)
+
+/*vdpaTypeToDriver translates vdpaTypes (as specified in the netDevice selectors)
+  to vdpa bus drivers*/
+var supportedVdpaTypes = map[types.VdpaType]string{
```

Comment:
Nitpick: `map[types.VdpaType]string{` - space missing befor the curly bracket (just for consistency with other definitions here)

Reply:
As a matter of fact, gofmt will remove that space automatically.
A quick grep:

```
$ grep -r "var.*map.*string{" . | wc -l
739
```
seems to indicate that it's the default way of writting it (most matches come from vendor, only [one](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin/blob/441e06f11bfafcf5f818975298943537a103b5a8/pkg/types/types.go#L69) in this codebase.

Grepping for the space only yields a (non gofmt-compliant?) code generator:
```
 $ grep -r "var.*map.*string {" .
./vendor/k8s.io/apimachinery/pkg/runtime/swagger_doc_generator.go:      s := fmt.Sprintf("var map_%s = map[string]string {\n", structName)
```